### ISSUE TITLE
Adding Runtime Permissions

### ISSUE THREAD

**Issue by samagra14** [scorelab/soundcom#38]
Currently, the app requires two permissions : 
1. Storage
2. Audio
These app permissions must be added as runtime permissions for API 23 and above. Currently, the app crashes as soon as we tap the mic button on devices using API 23 and above. This can be resolved by adding a runtime permission check just before recording and storage code blocks.

#### Are you interested to work on this ?
Yes.

**Comment by samagra14** [comment]
@charithccmc Please take a look at this issue.

**Comment by paras7735** [comment]
Is this issue taken?

**Comment by samagra14** [comment]
Yes, @paras7735 I have already implemented the permissions. I am just waiting for @Lasith-Niro to approve this issue and then I will send a PR. You may open another issue. For instance, there is some bug in the Stop button. Sometimes it doesn't work properly. Take a look at it. It might be an easy bug.

**Comment by paras7735** [comment]
I am really sorry.Sure ill open another issue.Thanks
